coop_account/models/account_move_line.py

```diff
@@ -138,12 +138,10 @@ def get_wrong_reconciliation_ml_data(self):
     @api.model
     def get_wrong_reconciliation_ml(self):
         wrong_move_lines = self
-        selected_journals = self.env['account.journal'].search([
-            ('name', 'not like', '%Chèques%'),
-            '|',
-            ('name', 'like', 'CCOOP - compte courant'),
-            ('name', 'ilike', '%cep%'),
-        ])
+        selected_journals = self.env['account.journal'].search(
+            [('name', 'not like', '%Chèques%')]
+        )
```

Comment:
I don't like this harcoded selected_journals.
Can you analyze how they're used and if it's possible to add a configuration somewhere, if necessary.. to avoid hardcoding it?

ie: a field in res.config.settings related to a company field maybe? or a field in the journal?

Reply:
@ivantodorovich  :I have already remove hardcoded journal from domain:
`('name', 'like', 'CCOOP - compte courant'),('name', 'ilike', '%cep%'),` but I am not getting reason to not consider journals that are start with `Chèques`  also i check in database journals start with `Chèques` are available so  I keep it as it is.

Reply:
I mean we should add a configuration somewhere.

The behaviour before/after this PR is going to be different, because now you removed a few journals that were hardcoded.

I agree to remove them, I don't like them being hardcoded, but this should be configurable somewhere. And since you're doing it, I think it also makes sense to remove the hardcoded %Chèques% search

Reply:
@ivantodorovich : I have added new boolean field on `account.journal`